code/modules/admin/sql_message_system.dm

```diff
@@ -219,6 +226,50 @@
 			browse_messages(target_ckey = target_ckey, agegate = TRUE)
 	qdel(query_find_edit_expiry_message)
 
+/proc/edit_message_severity(message_id, browse)
```

Comment:
Since the `if` check using browse doesn't really do anything, the browse argument is superfluous.